packages/components/psammead-brand/src/index.test.jsx

```diff
@@ -78,114 +78,76 @@ describe('Brand', () => {
-    });
-
-    it('should add extra props passed to the component', () => {
-      const { container } = render(
-        <Brand
-          product="Default Brand Name"
-          svgHeight={24}
-          maxWidth={280}
-          minWidth={180}
-          svg={svg}
-          url="https://www.bbc.co.uk/news"
-          backgroundColour={C_POSTBOX}
-          logoColour={C_WHITE}
-          data-brand="header"
-        />,
-      );
-
-      expect(container.querySelector('div').getAttribute('data-brand')).toEqual(
-        'header',
-      );
-    });
-    it('should render script, frontpage and skip to content links', () => {
-      const scriptLinkComponent = (
-        <ScriptLink
-          script={latin}
-          service="serbian"
-          href="https://www.bbc.com/serbian/lat"
-        >
-          Lat
-        </ScriptLink>
-      );
-
-      const skipLink = (
-        <SkipLink service="news" script={latin} href="#content">
-          Skip to content
-        </SkipLink>
-      );
-
-      const { container } = render(
-        <Brand
-          product="Default Brand Name"
-          svgHeight={24}
-          maxWidth={280}
-          minWidth={180}
-          svg={svg}
-          url="https://www.bbc.co.uk/news"
-          backgroundColour={C_POSTBOX}
-          logoColour={C_WHITE}
-          skipLink={skipLink}
-          data-brand="header"
-          scriptLink={scriptLinkComponent}
-        />,
-      );
-
-      const links = container.querySelectorAll('a');
-      expect(links).toHaveLength(3);
-
-      const frontpageLink = links[0];
-      expect(frontpageLink.getAttribute('href')).toEqual(
-        'https://www.bbc.co.uk/news',
-      );
-      expect(frontpageLink.textContent).toEqual('Default Brand Name');
-
-      const skipToContentLink = links[1];
-      expect(skipToContentLink.getAttribute('href')).toEqual('#content');
-      expect(skipToContentLink.textContent).toEqual('Skip to content');
+  it('should add extra props passed to the component', () => {
+    const { container } = render(
+      <Brand
+        product="Default Brand Name"
+        svgHeight={24}
+        maxWidth={280}
+        minWidth={180}
+        svg={svg}
+        url="https://www.bbc.co.uk/news"
+        backgroundColour={C_POSTBOX}
+        logoColour={C_WHITE}
+        data-brand="header"
+      />,
+    );
+
+    expect(container.querySelector('div').getAttribute('data-brand')).toEqual(
+      'header',
+    );
+  });
 
-      const scriptLink = links[2];
-      expect(scriptLink.getAttribute('href')).toEqual(
-        'https://www.bbc.com/serbian/lat',
-      );
-      expect(scriptLink.textContent).toEqual('Lat');
-    });
+  it('should render script, frontpage and skip to content links', () => {
+    const scriptLinkComponent = (
+      <ScriptLink
+        script={latin}
+        service="serbian"
+        href="https://www.bbc.com/serbian/lat"
+      >
+        Lat
+      </ScriptLink>
+    );
+
+    const skipLink = (
```

Comment:
Should this be called `skipLinkComponent`? Just for consistency with `scriptLinkComponent`.

Reply:
These weren't added in this PR, but any opportunity to improve naming consistency. 😄